include/boost/geometry/algorithms/detail/distance/range_to_geometry_rtree.hpp

```diff
@@ -113,7 +113,7 @@ class point_or_segment_range_to_geometry_rtree
                         <
                             typename selector_type::iterator_type
                         >::value_type,
-                    Strategy
+                Strategy
```

Comment:
This should probably stay indented, right?

Reply:
sure, IDE problem, thanks